src/app/auth/services/user.service.ts

```diff
@@ -62,8 +63,9 @@ const deleteUserAvatar = (): Promise<void> => {
 };
 
 const sendVerificationEmail = (): Promise<void> => {
-  const usersClient = SdkFactory.getInstance().createUsersClient();
-  return usersClient.sendVerificationEmail();
+  const usersClient = SdkFactory.getNewApiInstance().createUsersClient();
+  const token = localStorageService.get('xNewToken') || undefined;
```

Comment:
What do you think about to extract the value 'xNewToken' to a const in order to only change this in one place? I think that in `localStorageService` file it exists one object with that consts